pkgs/applications/networking/instant-messengers/nheko/default.nix

```diff
@@ -30,13 +29,13 @@
 
 mkDerivation rec {
   pname = "nheko";
-  version = "0.8.1";
+  version = "0.8.2-RC";
```

Comment:
If this is still an RC, now is probably a bit early to bump the nixpkgs version?

Reply:
Indeed, this is why this is just a draft.